### ISSUE TITLE
use outlets in a tabpanel

### ISSUE THREAD

**Issue by iirving** [IvyApp/ivy-tabs#29]
does ivy-tabs support using {{outlets}} within a tabpanel?

something like : 

{{#tabs.tabpanel "TabA"}}
     {{outlets "taba"}}
 {{/tabs.tabpanel}}

probably with the (nested) route for "taba" doing : 

  renderTemplate: function() {
          this.render({ outlet: 'taba' });
        },

or is that outside the functionally of the add-on?

**Comment by nbibler** [comment]
I can't say that I've tried to use the scenario you're describing. In our cases, the tabpanel content is local, model content. I'm guessing by using outlets you're trying to bind it more closely to the router and use the route models somehow as the tab content. In that case, it sounds less like a tab system and more like traditional site navigation.

**Comment by iirving** [comment]
yes! it was being used as navigation (routing nested models), rather than presenting sub-sections of the current model.

I ended up making my own tab component for that functionally, rather than Ivy-tabs otherwise more functional component.
